testcafe/runner-photos.js

```diff
@@ -1,17 +1,22 @@
 const createTestCafe = require('testcafe')
+const util = require('util')
+const exec = util.promisify(require('child_process').exec)
 
 async function runRunner() {
+  //init vrErrorMsg
+  process.env.vrErrorMsg = ''
+
   const tc = await createTestCafe()
   const runner = await tc.createRunner()
   const response = await runner
     .src([
       //Scenario that just upload photos, so we don't need to do it in every test.
       'testcafe/tests/photos/photos_start_upload_photos.js',
 
-      'testcafe/tests/photos/photos_crud.js',
-      'testcafe/tests/photos/create_full_album_scenario.js',
-      'testcafe/tests/photos/create_empty_album_scenario.js',
-
+      // 'testcafe/tests/photos/photos_crud.js',
+      // 'testcafe/tests/photos/create_full_album_scenario.js',
+      // 'testcafe/tests/photos/create_empty_album_scenario.js',
```

Comment:
Why are these commented out?

Reply:
I remove them for quicker local testing (as those tests don't have screenshots for now), and forgot to un-comment before commit!